connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -41,6 +41,7 @@ message Relation {
     Aggregate aggregate = 9;
     SQL sql = 10;
     LocalRelation local_relation = 11;
+    Deduplicate deduplicate = 13;
```

Comment:
12?
```suggestion
    Deduplicate deduplicate = 12;
```

Reply:
12 is take by `Sample` :) https://github.com/apache/spark/pull/38227

This line will cause merge conflict. Whether which PR is in first, it takes 12 and the other takes 13.